pkg/controller/kibana/config_reconcile.go

```diff
@@ -23,28 +23,43 @@ import (
 
 // Constants to use for the config files in a Kibana pod.
 const (
-	VolumeName        = "config"
-	VolumeMountPath   = "/usr/share/kibana/" + VolumeName
+	ConfigVolumeName                   = "elastic-internal-kibana-config-local"
```

Comment:
Would `shared` be more explicit than `local` to express how the volume is shared between init and main container?

Reply:
Naming has been chosen to be consistent with what is done with Elasticsearch:

https://github.com/elastic/cloud-on-k8s/blob/b6cb04d65574f18843dc3fcd93d50e5232009177/pkg/controller/elasticsearch/initcontainer/prepare_fs.go#L36-L40

I agree that `shared` would be more explicit but I'm not sure we want to do this refactoring for a patch release ?